.github/workflows/build-draft-docs.yml

```diff
@@ -1,38 +1,38 @@
 name: Build Draft Guidelines
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
This only allows manual triggering if it's part of the `main` branch, and runs the version from `main`

So this won't, for example, allow manual trigger of a modified workflow from a branch or PR. It also means that newer versions (in main) may not be compatible with older targets (e.g. old tagged releases) - because the tagged release expects to be run with the tagged version.

So this is not a catchall, but is mostly useful for folks working on their own repos, who pick up this file, and want to run for their local branches. As I did with the profiles work, since I hadn't committed recently.